snuba/perf.py

```diff
@@ -90,12 +92,24 @@ def write():
 
     time_start = time.time()
     if profile_process:
-        cProfile.runctx('process()', globals(), locals(), sort='cumulative')
+        filename = tempfile.NamedTemporaryFile(
```

Comment:
This adds some overhead to the timed block, but ultimately I don't think that really matters, since we shouldn't be using the execution time with the profiler running as a baseline for anything anyway, in my opinion.